### ISSUE TITLE
Port 9091 is opened by the client and conflicts with other component of my app which uses the same port

### ISSUE THREAD

**Issue by julianklas** [topfreegames/eventsgateway#7]
On this line:

https://github.com/topfreegames/eventsgateway/blob/e6c7bb6503b77592f6731c8a08819ca34cda4f28/metrics/metrics.go#L127

The port 9091 is opened and it conflicts with another functionality on my app that uses the same port.

I think that this port should be configurable if possible or I should be able to turn off this functionality. 

Right now I have to edit events gateway code to change the port number on the source code or my app just won't run.

**Comment by felipejfc** [comment]
Hi @julianklas, it make sense to have it configurable! This project was made during a hackathon so it doesn’t follow best practices in some parts of the code. Would you mind sending a PR?

**Comment by leeohaddad** [comment]
Hi @julianklas,

The port is configurable through the environment variable EVENTSGATEWAY_PROMETHEUS_PORT and the functionality can be disabled through the environment variable EVENTSGATEWAY_PROMETHEUS_DISABLED.

We analysed the possibility of configuring Viper to allow this to be set through config files also, but we arrived to the conclusion that this would not be a good idea now because it would break the functionality for people who are already using the environment variables, because of the prefix used by the hosting app.

We will add this to our to-do list anyway, to search for a better solution in the future and also to make a bigger refactor in this project. But, for now, I would recommend that you configure the behaviour using the env vars.
